org/pr/release-notes.ts

```diff
@@ -0,0 +1,37 @@
+import {warn, danger} from "danger";
+
+export default async () => {
+    const githubLabels = danger.github.issue.labels;
+    const modifiedFiles = danger.git.modified_files;
+    const pr = danger.github.pr;
+
+    // List of release notes files
+    const releaseNotesFiles: string[] = [
+        "RELEASE-NOTES.txt"
+    ];
+
+    // Skip if not on a release branch 
```

Comment:
```suggestion
    // Skip if not targeting a release branch 
```
Just to be clear that we only analyse PRs targeting the `release/*` branches, so PRs done after code freeze, but we won't check targets _from_ `release/*` (e.g. to `develop` or to any other branch)